### DISCUSSION TITLE
[LINUX] Compiling with NO OSD support (mame commandline only)

### DISCUSSION THREAD

**Discussion by cinnamonmatexfce** [mamedev/discussions#58] [Q&A] [answered]
I'm wondering if could be possible to compile MAME disabling UI.
I tried `0SD=0` but I got this error:
`mame-mame0257/scripts/genie.lua:1463: Unsupported value '0' for OSD`

Thanks

**Comment by rb6502** [comment, marked as answer]
No, the on-screen UI is pretty deeply integrated.  You can always bind Tab and ~ to something that isn't on your cabinet's control panel if that's why you're asking though.  Removing it for real is the territory of illegal commercial distributions of MAME so we obviously don't support that.  OSD= on the command line selects the Operating System Dependent layer (Windows-native or SDL or eventually Mac-native).
